app/controllers/yachts_controller.rb

```diff
@@ -6,8 +6,13 @@ def show
   end
 
   def new
+    @yacht = Yacht.new
   end
 
   def create
+     @yacht = Yacht.new(yacht_params)
+    if @yachts.save
```

Comment:
Should be `yacht` same as line 13

Reply:
Remember, before the `.save`, what does a yacht belongs_to ? A user
So we need an extra line
`@yacht.user = curent_user`

Meaning only a logged_in user should be able to access the form.
So we need to protect this controller action behind an `authenticate_user` 👍